README.md

```diff
@@ -43,6 +45,9 @@ Create a For Loop that outputs the following string for each piece of fruit in t
 
 ```js
 // write code here
+for (var i = 0; i < foods[0].length; i++) {
+  console.log("I want to eat a " + foods[0][i]);
+}
```

Comment:
This would only iterate through (part of) the first array inside of `foods`. To print out each fruit individually, you would need to nest another `for` loop inside of your existing one.